Procurement/ViewModel/StashViewModel.cs

```diff
@@ -296,49 +296,38 @@ void stashView_Loaded(object sender, RoutedEventArgs e)
                     case TabType.Currency:
                         var currencyStash = new CurrencyStash(currentTab.i, getUserFilter(string.Empty));
 
-                        item.Content = currencyStash;
-
-                        addContextMenu(item, currencyStash);
-
-                        tabsAndContent.Add(new TabContent(i - 1, item, currencyStash));
+                        CraftTabAndContent(item, currencyStash, i);
```

Comment:
This line is identical in all the cases, so we can move it after the switch statement.

Reply:
Done